lib/qtermwidget_version.h.in

```diff
@@ -0,0 +1,33 @@
+/*  Copyright (C) 2008 e_k (e_k@users.sourceforge.net)
```

Comment:
Please change this line to include your nickname, or use `LXQt team` like [1] in case you are not comfortable with leaving your name here.

[1] https://github.com/lxqt/lxqt-panel/blob/db7920b66b92d3e2c1b94d9d07e7a341940b8971/panel/panelpluginsmodel.h#L7

Reply:
OK done, the whole file is close to trivial anyway, I just copypasted qtermwidget.h as base.